content/outcomes/application-development/prioritizing-outcomes.md

```diff
@@ -0,0 +1,170 @@
+
+_But_, those features are _out of scope for this story_. The team should create new stories for those features with their own very explicit and tightly focused acceptance criteria. 
+
+#### Resources
+
+Attach anything else helpful explaining the story including mockups, sketches or user flows.
+
+## Describe the advantages of estimating work in terms of risk and complexity instead of time
+
+Simply put: [people are really bad at estimating based on time](https://en.wikipedia.org/wiki/Planning_fallacy). Yet, most projects, software or otherwise, insist on doing so. We recommend estimating tasks based on risk and complexity rather than time to completion.
+
+Agile software teams use **estimation** to understand the relative complexity of delivering a story. They do this because:
+
+* It helps the team understand and compare the development cost of new features, and make decisions accordingly.
+* It allows the team to get to a shared understanding of the story and the risks or unknowns they could encounter.
+* Over time, estimates can be used to predict how long it will take to deliver a particular feature or release.
+
+When discussing a story, usually in an [iteration or sprint planning meeting](https://tanzu.vmware.com/developer/practices/ipm/), teams will estimate the size of a story in terms of **story points**. Story points measure a story’s complexity and risk. Teams use different scales. For example:
+ 
+- A simple scale of 1, 2, or 3 points 
+- "T-shirt sizing" as Small (S), Medium (M) Large (L), Extra Large (XL)
+- The Fibonacci sequence: 1, 2, 3, 5, 8, 13 (or more) points
+
+We use a scale rather than time estimates because time estimates are known to be highly error prone. They do not account well for the uncertainty that teams often encounter when building software. Time estimates are especially prone to [planning fallacy](https://en.wikipedia.org/wiki/Planning_fallacy) and [optimism bias](https://en.wikipedia.org/wiki/Optimism_bias).
+
+**Relative complexity** means thinking about similar stories that the team has delivered in the past and making a relative estimate. This way, the team can build up a consistent, shared idea of what constitutes a small, medium or large-sized story.
+
+When team members give different estimates for a story, this can give rise to valuable conversations! Estimation prompts team members to talk about their assumptions and arrive at a shared understanding of what is involved in the work. Learn more in our [short guide to estimation](https://tanzu.vmware.com/developer/practices/ipm/#estimation).
+
+## Describe How Measuring Velocity Enables Teams to Predict Delivery
+
+> _"When is it going to be done?"_
+>
+> -- Your boss
+
+Every team responsible for a deliverable has been asked this question. 
+
+Software teams commonly need to gauge how long it takes to deliver a feature, or say whether a deadline will be met. Yet haven't we just described how we don't recommend estimating based on time of delivery?
```

Comment:
First sentence: Please rework this a bit. Here's why. The first part of the sentence reads, "Software teams commonly need to gauge how long it takes to deliver a feature,..." The second part of the sentence, "or say whether a deadline will be met."  My question: What do software teams do after gauging how long it's going to take to deliver a feature? Wouldn't they also have to provide a deadline at this point?

Reply:
Updated